src/app/screens/home/index.tsx

```diff
@@ -498,6 +554,43 @@ function Home() {
         />
       </Container>
       <BottomBar tab="dashboard" />
+
+      <BottomModal
+        visible={!!showDataCollectionAlert}
+        header=""
+        onClose={handleDataCollectionDeny}
+        overlayStylesOverriding={{
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
+        }}
+        contentStylesOverriding={{
+          bottom: 'initial',
+          borderRadius: 16,
+          maxWidth: 'calc(360px - 32px)',
```

Comment:
can you not use margin here?

Reply:
Actually, I can, I just needed to change the `width` from `100%` to `auto`.